src/kayenta/actions/creators.ts

```diff
@@ -38,6 +39,9 @@ export const updateMetricNanStrategy = createAction<{ id: string; strategy: stri
 export const updateMetricCriticality = createAction<{ id: string; critical: boolean }>(
   Actions.UPDATE_METRIC_CRITICALITY,
 );
+export const updateEffectSize = createAction<{ id: string; value: ICanaryMetricEffectSizeConfig }>(
+  Actions.UPDATE_EFFECT_SIZE,
+);
```

Comment:
I'm finally starting to understand redux a little bit and I don't hate it as much as I did when I didn't understand it at all.